public/index.html

```diff
@@ -97,6 +97,7 @@ <h1 id="version-info-id">v1.0.35</h1>
       <div id="feedback-text" class="feedback-text"></div>
       <canvas id="loading"></canvas>
       <canvas id="canvas"></canvas> <!--Main Canvas to Hold Stones-->
+      <div id="background"></div>
```

Comment:
make sure the structure is proper
background will the parent container and canvas, title, feedback etc. should comes under it. that is the best practice.